package.json

```diff
@@ -16,6 +16,8 @@
     "test": "mocha -r ts-node/register ./tests/**/*.test.ts"
   },
   "dependencies": {
+    "@types/amqp-connection-manager": "^2.0.10",
+    "amqp-connection-manager": "^3.2.1",
     "amqplib": "^0.5.5",
```

Comment:
Remove amqplib

Reply:
Also I'm seeing rules-js as a dependancy. I'm sure that was my fault at some point but could you remove that while you are at it too.

Reply:
@daniellmorris I pulled out rules-js, but couldn't pull out amqplib completely because we still need access to some of the classes in there (and they're not exposed by amqp-connection-manager). We can discuss if ya want.